unity/Hello_World/Packages/manifest.json.backup3

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
what are these "backup3" files? Do they really need to be committed or are they things that should be ignored?